src/commands/openapi/inspect.ts

```diff
@@ -108,68 +108,71 @@ function buildFullReport(analysis: Analysis, definitionVersion: string, tableBor
   const report: string[] = ['Here are some interesting things we found in your API definition. 🕵️', ''];
 
   // General API definition statistics
-  Object.entries(analysis.general).forEach(([, info]) => {
-    let msg: string;
+  report.push(
+    ...(Object.entries(analysis.general || {})
+      .map(([, info]) => {
+        if (Array.isArray(info.found)) {
+          if (!info.found.length) return false;
+
+          const highlightedData = info.found.map(d => chalk.yellow(d));
+          if (info.found.length > 1) {
+            const pluralName = pluralize(info.name.toLowerCase(), info.found.length);
+            return `You are using ${chalk.bold(info.found.length)} ${pluralName} throughout your API: ${new Intl.ListFormat('en').format(highlightedData)}`;
+          }
+
+          return `You are using a single ${info.name.toLowerCase()} throughout your API: ${highlightedData[0]}`;
+        } else if (info.found > 1) {
+          let msg: string;
+          msg = `You have a total of ${chalk.bold(info.found)} ${pluralize(info.name.toLowerCase(), info.found)} in your API.`;
+          if (info.found > 100) {
+            msg += ` ${chalk.cyanBright('Wow! 🤯')}`;
```

Comment:
iffy on the emoji but thought it might be fun for the true Big API sickos if this stood out a bit more. also might bump this 100 operation threshold up to maybe 200 because after this week of debugging failed API uploads 100 operations in a single file seems so small

<img width="748" alt="Screenshot 2024-12-20 at 9 44 03 PM" src="https://github.com/user-attachments/assets/e48ced22-ca83-4b16-87ad-57b84da5a505" />

Reply:
obsessed with this